src/controller/CHIPDeviceController.cpp

```diff
@@ -3062,28 +3041,9 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
         request.breadcrumb.Emplace(breadcrumb);
 
         CHIP_ERROR err = CHIP_NO_ERROR;
-        GeneralCommissioning::Attributes::SupportsConcurrentConnection::TypeInfo::Type supportsConcurrentConnection;
-        supportsConcurrentConnection = params.GetSupportsConcurrentConnection().Value();
         ChipLogProgress(Controller, "SendCommand kWiFiNetworkEnable, supportsConcurrentConnection=%d",
-                        supportsConcurrentConnection);
-        if (supportsConcurrentConnection)
-        {
-            err = SendCommand(proxy, request, OnConnectNetworkResponse, OnBasicFailure, endpoint, timeout);
-        }
-        else
-        {
-            // Concurrent Connections not allowed. Send the ConnectNetwork command but do not wait for the
-            // ConnectNetworkResponse on the Commissioning network as it will not be present. Log the expected timeout
-            // and run what would have been in the onConnectNetworkResponse callback.
-            err = SendCommand(proxy, request, NonConcurrentNetworkResponse, NonConcurrentTimeout, endpoint, NullOptional);
-            if (err == CHIP_NO_ERROR)
-            {
-                // As there will be no ConnectNetworkResponse, it is an implicit kSuccess so a default report is fine
-                CommissioningDelegate::CommissioningReport report;
-                CommissioningStageComplete(CHIP_NO_ERROR, report);
-                return;
-            }
-        }
+                        params.GetSupportsConcurrentConnection().Value());
```

Comment:
So now we are reading the "supports concurrent connection" bit just so we can log it here?

Reply:
Yes, with the change to always send ConnectNetworkResponse(Success) on BLE prior to closing the commissioning channel and starting the operational channel. However I can see additional optimisation that could be done by the controller if knowing that this is a non-concurrent device.